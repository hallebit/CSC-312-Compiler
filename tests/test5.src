if ((fun (y:int) : int -> y + 5) 3) <= 4 then
 let x:int = 0 in x * 
    5 else 15 