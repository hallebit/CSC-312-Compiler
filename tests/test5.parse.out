(if (((fun y -> (y + 5)) 3) <= 4) then (let x = 0 in (x * 5)) else false)